src/backend/base/langflow/api/v1/chat.py

```diff
@@ -155,6 +156,7 @@ async def build_flow(
     queue_service: Annotated[JobQueueService, Depends(get_queue_service)],
     flow_name: str | None = None,
     settings_service: Annotated[SettingsService, Depends(get_settings_service)],
+    event_delivery: EventDeliveryType = EventDeliveryType.STREAMING,
```

Comment:
The default in settings is `polling`
```suggestion
    event_delivery: EventDeliveryType = EventDeliveryType.POLLING,
```